.evergreen/build_snapshot_rpm.sh

```diff
@@ -40,7 +40,7 @@ done
 
 package=mongo-c-driver
 spec_file=../mongo-c-driver.spec
-spec_url=https://src.fedoraproject.org/rpms/mongo-c-driver/raw/master/f/mongo-c-driver.spec
+spec_url=https://src.fedoraproject.org/rpms/mongo-c-driver/raw/rawhide/f/mongo-c-driver.spec
```

Comment:
Ah, I assume this was due to Fedora's branch rename: https://fedoraproject.org/wiki/Changes/GitRepos-master-to-main#Summary

Reply:
Correct.  The rename was actually `master` -> `main`.  However, since semantically we want "the latest", `rawhide` is the proper name to track.  In reality, the Fedora source hosting service automatically aliases rawhide to the default Git branch (which is now `main`).